### PR TITLE
[RPS-492] Fix access to S3 assets folder

### PR DESCRIPTION
**Pull request by ziemsky**
Members of group associated with policy AssetsPolicy can now browse
entire S3 bucket but can perform actions only within folder 'assets',
unless another policy grants them wider rights.

Also updated readme with more info on first-time setup of AWS CLI
client and a clarification of 'make stack' args.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,6 +78,13 @@ Everything you need will be installed locally. Simply run:
 make init
 ```
 
+Perform a one-off configuration of your local Amazon CLI client by executing:
+```bash
+ansible/.venv/bin/aws --profile nhsd-profile-name configure
+```  
+This will ask you to provide your AWS access key id and access key value, AWS region
+(`eu-west-1`) and preferred [format of output generated by the client](https://docs.aws.amazon.com/cli/latest/userguide/controlling-output.html).
+It then saves those details in `~/.aws/configuration` and `~/.aws/credentials`. The name of the profile (``)
```
Comment (jacsamell) [inline README.md]: 👍

Comment (jacsamell) [reply README.md]: Is this sentence finished? 😃

File: ansible/roles/foundations/aws_permissions/files/policies.yml [hunk 2]
```diff
@@ -51,42 +51,32 @@ Resources:
...
-              StringEquals:
-                s3:delimiter: /
-                s3:prefix:
-                  - ""
-                  - assets
-                  - assets/
-          # Allow access to assets subfolder
-          - Sid: AccessToAssetsSubfolder
+          # Allow access to 'assets' subfolder
+          - Sid: AssetsFolderContentOperations
```
Comment (jacsamell) [inline ansible/roles/foundations/aws_permissions/files/policies.yml:71]: Simples 😀

**Review by jacsamell — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ziemsky) [reply README.md]: Good spot!

Unusually, my IDE was giving me some problems when editing Markdown, for some reason,  and it seems I've lost some of the content.

Now fixed.

**Pull request merged**